sched/distributor/poolDistributor.go

```diff
@@ -0,0 +1,63 @@
+package distributor
+
+import (
+	"github.com/scootdev/scoot/sched"
+	cm "github.com/scootdev/scoot/sched/clustermembership"
+)
+
+//
+// Pool Distributor will assign work to any
+// Unscheduled Node in the cluster.
+//
+// While a Node is doing work no new tasks will
+// be sent to it.  Blocks until a node becomes free
+//
+// Static Cluster for now
+//
+type PoolDistributor struct {
+	freeCh chan cm.Node
+}
+
+//
+// Creates a Pool Distributor, assumes
+// all nodes in the cluster are free upon creation
+//
+// If the cluster has 0 nodes in it returns nil
+//
+func NewPoolDistributor(cluster cm.Cluster) *PoolDistributor {
```

Comment:
I should've asked earlier, but what do we gain by having the interface cm.Cluster vs just passing around []cm.Node? Are we planning to enhance cm.Cluster in any way?